db/people.yml

```diff
@@ -546,3 +546,31 @@ sbanerjee:
   name: Soham Banerjee
   position: Graduate Student
   orcid_id: 0000-0001-9271-493X
+stao:
+  aka:
+    - Songsheng Tao
+    - Tao, Songsheng
+    - S. Tao
+    - Tao, S.
+  avatar: ''
```

Comment:
Please add this.  What we need is for you to upload a picture you want of yourself to GitHub, then following the template of others, put the URL to that picture here.  The GitHub picture will then be used as your picture also in the group.  Make it a picture you are proud of!